src/peergos/server/Start.java

```diff
@@ -86,9 +90,87 @@
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    System.exit(1);
+                }
+            },
+            Collections.emptyList()
+    );
+
+    public static final Command POSTSTRAP = new Command("poststrap",
+            "The final step of bootstrapping a new peergos network, which must be run after it is started",
+            args -> {
+                try {
+                    Crypto crypto = Crypto.initJava();
+                    // recreate peergos user and pki keys
+                    String testpassword = args.getArg("peergos.password", "testpassword");
+                    String pkiUsername = "peergos";
+                    UserWithRoot peergos = UserUtil.generateUser(pkiUsername, testpassword, crypto.hasher, crypto.symmetricProvider,
+                            crypto.random, crypto.signer, crypto.boxer, UserGenerationAlgorithm.getDefault()).get();
+
+                    SigningKeyPair peergosIdentityKeys = peergos.getUser();
+                    PublicKeyHash peergosPublicHash = ContentAddressedStorage.hashKey(peergosIdentityKeys.publicSigningKey);
+                    PublicSigningKey pkiPublic = args.hasArg("pki.public.key") ?
+                            PublicSigningKey.fromByteArray(
+                                    Files.readAllBytes(Paths.get(args.getArg("pki.public.key")))) :
+                            IpfsCoreNode.PKI_PUBLIC_KEY;
+                    SecretSigningKey pkiSecretKey = SecretSigningKey.fromCbor(CborObject.fromByteArray(
+                            Files.readAllBytes(Paths.get(args.getArg("pki.secret.key")))));
+                    SigningKeyPair pkiKeys = new SigningKeyPair(pkiPublic, pkiSecretKey);
+                    PublicKeyHash pkiPublicHash = ContentAddressedStorage.hashKey(pkiKeys.publicSigningKey);
+                    int webPort = args.getInt("port");
+                    NetworkAccess network = NetworkAccess.buildJava(new URL("http://localhost:" + webPort)).get();
+
+                    // sign up peergos user
+                    UserContext context = UserContext.ensureSignedUp(pkiUsername, testpassword, network, crypto).get();
+                    context.addNamedOwnedKeyAndCommit("pki", pkiPublicHash).get();
+                    // write pki public key to ipfs
+                    network.dhtClient.putSigningKey(peergosIdentityKeys.secretSigningKey
+                            .signatureOnly(pkiKeys.publicSigningKey.serialize()), peergosPublicHash, pkiKeys.publicSigningKey).get();
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    System.exit(1);
+                }
+            },
+            Collections.emptyList()
+    );
+
     public static final Command LOCAL = new Command("local",
             "Start an ephemeral Peergos Server and CoreNode server",
             args -> {
+                BOOTSTRAP.main(args);
                 args.setIfAbsent("domain", "localhost");
```

Comment:
TODO: stop modifying ``args`` and passing it to other  things that modify it.

Reply:
a future pr